py/build.gradle

```diff
@@ -3,6 +3,13 @@ apply from: "$rootDir/gradle/utils.gradle"
 ext {
     FS = File.separator
     FPS = File.pathSeparator
+
+    def command = ["python", "py/pythonVersion.py"]
```

Comment:
@jakubhava that one works fine, I was having problems with the `-c` option in my case.